src/FHIR/R4/FHIRResourceContainer.php

```diff
@@ -3468,445 +3468,153 @@ public function __toString()
-        }
-        if (isset($this->ResearchSubject)) {
-            return $this->ResearchSubject;
-        }
-        if (isset($this->RiskAssessment)) {
-            return $this->RiskAssessment;
-        }
-        if (isset($this->RiskEvidenceSynthesis)) {
-            return $this->RiskEvidenceSynthesis;
-        }
-        if (isset($this->Schedule)) {
-            return $this->Schedule;
-        }
-        if (isset($this->SearchParameter)) {
-            return $this->SearchParameter;
-        }
-        if (isset($this->ServiceRequest)) {
-            return $this->ServiceRequest;
-        }
-        if (isset($this->Slot)) {
-            return $this->Slot;
-        }
-        if (isset($this->Specimen)) {
-            return $this->Specimen;
-        }
-        if (isset($this->SpecimenDefinition)) {
-            return $this->SpecimenDefinition;
-        }
-        if (isset($this->StructureDefinition)) {
-            return $this->StructureDefinition;
-        }
-        if (isset($this->StructureMap)) {
-            return $this->StructureMap;
-        }
-        if (isset($this->Subscription)) {
-            return $this->Subscription;
-        }
-        if (isset($this->Substance)) {
-            return $this->Substance;
-        }
-        if (isset($this->SubstanceNucleicAcid)) {
-            return $this->SubstanceNucleicAcid;
-        }
-        if (isset($this->SubstancePolymer)) {
-            return $this->SubstancePolymer;
-        }
-        if (isset($this->SubstanceProtein)) {
-            return $this->SubstanceProtein;
-        }
-        if (isset($this->SubstanceReferenceInformation)) {
-            return $this->SubstanceReferenceInformation;
-        }
-        if (isset($this->SubstanceSourceMaterial)) {
-            return $this->SubstanceSourceMaterial;
-        }
-        if (isset($this->SubstanceSpecification)) {
-            return $this->SubstanceSpecification;
-        }
-        if (isset($this->SupplyDelivery)) {
-            return $this->SupplyDelivery;
-        }
-        if (isset($this->SupplyRequest)) {
-            return $this->SupplyRequest;
-        }
-        if (isset($this->Task)) {
-            return $this->Task;
-        }
-        if (isset($this->TerminologyCapabilities)) {
-            return $this->TerminologyCapabilities;
-        }
-        if (isset($this->TestReport)) {
-            return $this->TestReport;
-        }
-        if (isset($this->TestScript)) {
-            return $this->TestScript;
-        }
-        if (isset($this->ValueSet)) {
-            return $this->ValueSet;
-        }
-        if (isset($this->VerificationResult)) {
-            return $this->VerificationResult;
-        }
-        if (isset($this->VisionPrescription)) {
-            return $this->VisionPrescription;
-        }
-        if (isset($this->Parameters)) {
-            return $this->Parameters;
-        }
-        return null;
+        return $this->Account
```

Comment:
@openemr/admins this was an interesting refactor. Rector would only do one of these at a time, so I had to run it in a loop for however many of these items there are. Then I realized that the whole thing looks like generated code. For generated code, is there a way to keep it updated / keep regenerating it on update?

Reply:
not sure about that but think this won't be regenerated again since we'll replace with the next version of FHIR five

Reply:
very interesting line of code :)
if do regenerate it, just will likely need to do the same thing :)